modrob0:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 250
  arm_position_controller:
    type: effort_controllers/JointGroupPositionController
    joints:
      - joint0
      - joint1
      - joint2
    joint0:
      pid: {p: 5000.0, i: 0.0, d: 112.0}
    joint1:
      pid: {p: 10000.0, i: 0.0, d: 0.0}
    joint2:
      pid: {p: 700.0, i: 0.0, d: 0.0}
  gripper_position_controller:
    type: effort_controllers/JointGroupPositionController
    joints:
      - hand_to_finger1
      - hand_to_finger2
    hand_to_finger1: 
      pid: {p: 1000.0, i: 0.0, d: 0.0}
    hand_to_finger2: 
      pid: {p: 1000.0, i: 0.0, d: 0.0}
